Solutions/Marain.UserNotifications.Specs/GlobalSuppressions.cs

```diff
@@ -10,4 +10,4 @@
 [assembly: System.Diagnostics.CodeAnalysis.SuppressMessage(
     "StyleCop.CSharp.OrderingRules",
     "SA1633:Using directives should be ordered alphabetically by namespace",
-    Justification = "We need this until https://github.com/SpecFlowOSS/SpecFlow/issues/1828 is fixed")]
+    Justification = "We need this until https://github.com/SpecFlowOSS/SpecFlow/issues/1828 is fixed")]
```

Comment:
Ah. That issue has in fact been fixed, and by updating to the latest Corvus libraries you will now have picked up that SpecFlow fix, so you can remove this suppression now. And if it's the only suppression left in the file, we can remove the file entirely.

You should also search the whole solution for similar suppressions: there might be more doing the same thing in other projects that don't show up in this PR because this one only showed up because of the change in line ending.